crypto/bio/bss_dgram.c

```diff
@@ -2809,7 +2809,9 @@ static void get_current_time(struct timeval *t)
     t->tv_usec = ((int)(now_ul % 10000000)) / 10;
 # else
     if (gettimeofday(t, NULL) < 0)
-        perror("gettimeofday");
+        ERR_raise_data(ERR_LIB_SYS, errno,
+                       "calling gettimeofday");
```

Comment:
Nit: doesn't fit on one 80-column line?

Reply:
This will actually be removed by #19082, so I probably don't need to even include it.